packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/add.controller.js

```diff
@@ -28,7 +29,7 @@ export default class ColdArchiveConfigurationController {
       },
       createOrLinkedMode: null,
     };
-
+    this.trackPage(COLD_ARCHIVE_TRACKING.ADD.MAIN);
```

Comment:
**nipick:** Dont see an `atInternet: { ignore: true }` in the state definition. So this will track the page twice.
Please try and use default tracking applied on every page.
Let me know if you need more info regarding default tracking.

Reply:
I've added `atInternet: { ignore: true }` to `*.routing.js` files of cold-archive module.

If you'll be available, I would like to schedule a small session with you to have a better understanding on the implementation of the tracking in the Manager.

Your feedback concerning the default tracking will be applied in the refactoring task MANAGER-10677 that will be scheduled for the next sprint.